### PR TITLE
Bug 1777038: Add alerting rules for subnet allocations

### PR DESCRIPTION
**Pull request by abhat**
/hold
/assign

### CODE REVIEW

**Comment by abhat** [comment]
Depends on github.com/ovn-org/ovn-kubernetes/pull/360

**Comment by abhat** [comment]
@tssurya @squeed PTAL.

**Comment by abhat** [comment]
@squeed, @tssurya can you folks PTAL?

File: bindata/network/ovn-kubernetes/alert-rules-control-plane.yaml [hunk 1]
```diff
@@ -48,3 +48,21 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: V4SubnetAllocationThresholdExceeded
+      annotations:
+        message: |
+          More than 80% of v4 subnets available to assign to the nodes are allocated
```
Comment (squeed) [inline bindata/network/ovn-kubernetes/alert-rules-control-plane.yaml:54]: You should add templating to the message (see openshift-sdn/alert-rules.yaml) that displays the actual percentage allocated.

Comment (abhat) [reply bindata/network/ovn-kubernetes/alert-rules-control-plane.yaml]: Done.

**Pull request merged**